.circleci/config.yml

```diff
@@ -10,93 +10,81 @@
 # DOCKER_PASS
 
 version: 2
-
-# This is not part of circleci, but it lets us not repeat ourselves.
-STUFF:
-  python_image: &python_image
-    image: python:3.8-alpine
-
-  # it's useful to not build branches that contain slashes,
-  # if it is later decided to map branches to docker tags.
-  main_filters: &main_filters
-    branches:
-      ignore:
-        - /\//
-
 jobs:
-  lint:
-    working_directory: /app
+  full:
     docker:
-      - *python_image
-    steps:
-      - checkout
-      - run: apk add gcc linux-headers libc-dev
-      - run: pip install black==19.10b0 && pip install flake8==3.7.9
-      - run: black --diff .
-      - run: flake8 .
-  test:
-    working_directory: /app
-    docker:
-      - *python_image
+      - image: circleci/python
+        user: root
```

Comment:
Do you need the `circleci/python` image if you aren't doing any Python commands in the Circle CI shell? Looks like all the Python work is happening within the `docker/Dockerfile-prod/` image

Reply:
Correct. Will switch this to a more bare bones image.